apluslms_roman/cli.py

```diff
@@ -67,10 +67,11 @@ def __init__(self, *args, callback_dest=None, callback_subparser_defaults=None,
 
         if callback_dest is None:
             callback_dest = 'action'
-        self.register('action', 'parsers', partial(CallbackSubParsersAction, callback_dest=callback_dest))
+        self.register('action', 'parsers',
+            partial(CallbackSubParsersAction, callback_dest=callback_dest))
         self._callback_dest = callback_dest
         self._callback_subparsers = None
-        self._callback_subparser_defaults = callback_subparser_defaults.copy() if callback_subparser_defaults is not None else {}
+        self._callback_subparser_defaults = (callback_subparser_defaults or {}).copy()
```

Comment:
Old code is 50% faster, but I get that the old code is also bit hard to read. Suggestion:
```
if callback_subparser_defaults is not None:
  self._callback_subparser_defaults = callback_subparser_defaults.copy()
else:
  self._callback_subparser_defaults = {}
```